src/js/components/navigation-bar.jsx

```diff
@@ -121,7 +121,7 @@ var NavigationBar = React.createClass({
                         <label htmlFor="scene-graph-input" className="mf-text">Graph Viewer</label>
                         <input type='text' ref='name' id="scene-graph-input" className='form-control'
                                placeholder='Room ID'/>
-                        <button type='submit' className='btn btn-dark'>set Room</button>
+                        <button type='submit' className='btn btn-dark'>Set Room</button>
```

Comment:
JIRA CORE-122 was CORE-114 something else?

Reply:
Yeah looks like I've added a redundant ticket.  I've deleted one of them.